tests/h/schemas/forms/admin/group_test.py

```diff
@@ -162,18 +183,27 @@ def group_data(factories):
     return {
         "name": "My Group",
         "group_type": "open",
-        "creator": factories.User().username,
+        "creator": "valid_creator",
         "description": "Lorem ipsum dolor sit amet consectetuer",
-        "organization": "__default__",
+        "organization": org.pubid,
         "scopes": ["http://www.foo.com", "https://www.foo.com"],
         "enforce_scope": True,
     }
 
 
 @pytest.fixture
-def user_svc(pyramid_config):
+def user_svc(pyramid_config, factories):
     svc = mock.create_autospec(UserService, spec_set=True, instance=True)
     pyramid_config.register_service(svc, name="user")
+
+    def fetch(username, authority):
+        if "invalid" in username:
+            return False
+
+        return factories.User()
```

Comment:
Having this fail everytime caused non-isolated tests where multiple things were failing at the same time. Our tests should now always pass unless we set a username with "invalid" in it.